Makefile

```diff
@@ -0,0 +1,71 @@
+# A simple Makefile intended to facilitate repetitive tasks
+
+# Dependencies installation
+.PHONY: dep-install
+dep-install:
+	@echo "#### Installing dependencies ####"
+	pip install numpy
+	pip install scipy
+	pip install scikit-learn
+	# Used to run the examples
+	pip install matplotlib
+	# Package needed to run tests in an automate way
+	pip install nose
+	@echo "#### All dependencies were installed successfully ####"
+
+# Test related commands
+NOSETESTS ?= nosetests
+
+.PHONY: test-all
+test-all:
+	nosetests
+
+# Examples related commands
+.PHONY: run-poisson
+run-poisson:
+	python examples/api/plot_poisson.py
+
+.PHONY: run-poisson2
```

Comment:
I see your point. These packages aren't necessary to make Pyglmnet works. They are useful to run the examples only. I gonna remove them and the test commands.